ui/tests/acceptance/task-logs-test.js

```diff
@@ -62,10 +62,7 @@ module('Acceptance | task logs', function (hooks) {
 
   test('the stdout log immediately starts streaming', async function (assert) {
     await TaskLogs.visit({ id: allocation.id, name: task.name });
-    const node = server.db.nodes.find(allocation.nodeId);
-    const logUrlRegex = new RegExp(
-      `${node.httpAddr}/v1/client/fs/logs/${allocation.id}`
-    );
+    const logUrlRegex = new RegExp(`/v1/client/fs/logs/${allocation.id}`);
```

Comment:
Also related to the task log streaming tests that all got management tokens, but this test just wants to know "is streaming happening right away" so rather than invoke ACL policy stuff in an otherwise kind of lean test, I just changed the local url regex.